script/Ciel/Day 13/13_01_01_CIEL13_3B.txt

```diff
@@ -1544,7 +1659,8 @@ I don't know what it feels like to be in so much pain that I'd rather die.
 [sha:b513079a0e4ed270fbc5a2c0c4649cb1e8ab3945]{
 -- Page 236, Offset 33857.
 -- 生きている以上、生を苦しく思う事はあるだろうけど、心の底から死を望む事はない。
-There have been times in life where I've struggled to keep living on, but I've never truly wished for death from the bottom of my heart.
+There have been times in life where I've struggled to keep on going,
+ but I've never truly wished for death from the bottom of my heart.
 }
 [sha:851d38e657e6ee9519c45613b118e82d4a6cd203]{
 -- Page 236, Offset 33858.
```

Comment:
33858
"hope and salvation" comes too close to "hope of salvation", so I'd rather it be split up a bit.

>death is her sole wish, her sole salvation.